Source/WebKit/UIProcess/WebPreferences.cpp

```diff
@@ -77,6 +77,34 @@ WebPreferences::~WebPreferences()
     ASSERT(m_pages.computesEmpty());
 }
 
+const Vector<RefPtr<API::Object>>& WebPreferences::experimentalFeatures()
+{
+    static auto experimentalFeatures = NeverDestroyed([]() {
+        Vector<RefPtr<API::Object>> result;
+        for (auto& object : features()) {
+            API::FeatureStatus status = static_pointer_cast<API::Feature>(object)->status();
+            if (status == API::FeatureStatus::Developer || status == API::FeatureStatus::Testable || status == API::FeatureStatus::Preview || status == API::FeatureStatus::Stable)
+                result.append(object);
+        }
+        return result;
+    }());
+    return experimentalFeatures;
+}
+
+const Vector<RefPtr<API::Object>>& WebPreferences::internalDebugFeatures()
+{
+    static auto internalDebugFeatures = NeverDestroyed([]() {
+        Vector<RefPtr<API::Object>> result;
+        for (auto& object : features()) {
+            API::FeatureStatus status = static_pointer_cast<API::Feature>(object)->status();
+            if (status == API::FeatureStatus::Unstable || status == API::FeatureStatus::Testable)
```

Comment:
I think this should be:

`API::FeatureStatus::Unstable || API::FeatureStatus::Internal`

Reply:
That's right, good catch!